docs/reference/transform/apis/start-transform.asciidoc

```diff
@@ -56,7 +57,16 @@ required privileges on the source and destination indices, the
 
 `<transform_id>`::
 (Required, string)
-include::{es-repo-dir}/rest-api/common-parms.asciidoc[tag=transform-id]
+Identifier for the {transform}.
+
+[[start-transform-query-parms]]
+== {api-query-parms-title}
+
+`timeout`::
+(Optional, time)
+Period to wait for a response. If no response is received before the timeout
+expires, the request fails and returns an error.
```

Comment:
The default is `30s`.

`timeout` applies to all API's, except GET, so this is a common parameter.

Reply:
Thanks!  I've updated the REST API specs in 646656a2c237b8a8bd7d1e0e2f070822108a3ed2. Could you please confirm that's the right list? Then I'll make the same changes in the docs